package.json

```
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "nyc --reporter=text --reporter=lcov --check-coverage --branches 100 --statements 100 --lines 100 mocha",
     "lint": "./node_modules/.bin/eslint .",
+    "debug": "node ./debug/debug.js",
```

why not just `npm start` ?

```suggestion
    "start": "node ./debug/debug.js",
```